src/stories/1-Button.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
- [ ] To keep things organized please keep all stories in the same folder as the component.

Reply:
Sorry about this gavin, didn't purposely create these they were just generated from running the storybook install commands that's why I initially put that stuff in the gitignore cause I had no clue what was going on lol 😅 should I just remove these for now? I'm not making any use of the storybook thing atm anyway just wanted to see what it was

Reply:
@marcoseoane You can certainly remove all the storybook work for now and I can create a separate task for that work. However, it would be nice to have a storybook for the Button now. I'll let you decide what you want to do and if you are interested in creating the storybook then please send me a DM so I can explain it better to you! :)